backend/benefit/applications/api/v1/serializers.py

```diff
@@ -67,21 +67,25 @@
 from users.utils import get_company_from_request, get_request_user_from_context
 
 
-class ApplicationBasisSerializer(serializers.ModelSerializer):
+class DynamicFieldsModelSerializer(serializers.ModelSerializer):
     """
-    Only the unique identifier is exposed through the API.
+    A ModelSerializer that takes an additional `fields` argument that
+    controls which fields should be displayed.
```

Comment:
..."and `exclude_fields` parameter argument that controls which fields should not be displayed." or something similar perhaps?

Reply:
good point, added the suggested text